src/Wrapper.js

```diff
@@ -262,15 +262,10 @@ class Wrapper extends React.Component {
       )
     }
 
-    // The Settings need the wrapper
-    if (!wrapper) {
-      return <LoadingApps />
-    }
-
     if (instanceId === 'settings') {
       return (
         <Settings
-          cache={wrapper.cache}
+          cache={wrapper ? wrapper.cache : null}
```

Comment:
I think we can remove this entirely now, since the cache isn't being used at all.